### DISCUSSION TITLE
ORMLite not working with Jakarta Persistence API.

### DISCUSSION THREAD

**Discussion by infinite-dev22** [j256/ormlite-core#284] [Q&A]
This is my sample class entity using Jakarta.
```
import jakarta.persistence.*;

@Entity
public class Supplier {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  private int id;

  @Column(nullable = false)
  private String name;

  ....
  // Getters and Setters.
  ....
}
```

When I run the project in IDEA, ORMLite throws an ```SQLException``` of ```No Database Fields defined```

Does ORMLite support Jakarta Persistence API or it still uses Java Persistence API.

Thanks in advance.

**Comment by j256** [comment]
Sorry it does not support Jakarta Persistence API.

**Reply by infinite-dev22** [reply]
Can it's support be included, I don't want to use Java Persistence API as it's outdated.
